### ISSUE TITLE
Missing language code `enm`

### ISSUE THREAD

**Issue by jack-mil** [rockrider69/service.LanguagePreferenceManager#11]
Some (anime) includes subtitle tracks with the language code `enm`. I (erroneously) thought this plugin would match any arbitrary language code that subtitles might be tagged with. Looks like a language has to be explicitly supported in the collection of language codes `langcodes.py`. I'm attempting to look at what info would be needed here. There is only an ISO-639-2 (three-letter) code. And I am not sure what podnapsi column is (used for).

>  info <general>: [Language Preference Manager]: Custom audio prefs: lang code enm not found in db! Please report this

TL;DR: I want to match tracks tagged with `enm` language code.

Example media info output: 
```json
{
"@type": "Text",
"@typeorder": "1",
"StreamOrder": "2",
"ID": "3",
"UniqueID": "8646590641823328841",
"Format": "ASS",
"CodecID": "S_TEXT/ASS",
"Duration": "1429.310000000",
"BitRate": "161",
"FrameRate": "0.232",
"FrameCount": "332",
"ElementCount": "332",
"Compression_Mode": "Lossless",
"StreamSize": "28940",
"Language": "en",
"Default": "Yes",
"Forced": "No"
},
{
"@type": "Text",
"@typeorder": "2",
"StreamOrder": "3",
"ID": "4",
"UniqueID": "4877982355693749142",
"Format": "ASS",
"CodecID": "S_TEXT/ASS",
"Duration": "1429.310000000",
"BitRate": "161",
"FrameRate": "0.232",
"FrameCount": "332",
"ElementCount": "332",
"Compression_Mode": "Lossless",
"StreamSize": "28940",
"Title": "Honorifics",
"Language": "enm",
"Default": "No",
"Forced": "No"
}
```

**Comment by jack-mil** [comment]
I adding entry, and it worked to select the desired subtitle track, but I am not sure if there are values or edge cases that need to be handled in the context of the addon.

I am using custom subtitle preference `enm > eng`. Not sure how (or if it is even desired) to have this "language" appear in the list of regular Language names e.g. "French, English, German"

- `langcodes.py`
    ```py
    LANGUAGES      = (
    # Full Language name[0]     podnapisi[1]      ISO 639-1[2]   ISO 639-1 Code[3]   Script Setting Language[4]   localized name id number[5]
    # ...
    ("English (Middle)" ,          "-3",              "",              "enm",                   "101",              30212  ),
    # ...
    )
```

**Comment by rockrider69** [comment]
Hello!
as you suggest the best is to add a new language line in the table with name "English (Middle)".
But to limit the changes in other modules, it is prefered to add it at the end of the table after Vietnamese in fact, not to mess with Script Setting id's and localized  name id's.
You are the 2nd userto ask for enm code support, I will add this in my to do list for LPM 1.0.5 ;)

**Comment by rockrider69** [comment]
@jack-mil 
can you please try LPM 1.0.5 from my github, to see if everything is ok from your side ?

**Comment by jack-mil** [comment]
works for me!
Thanks

**Comment by rockrider69** [comment]
Fixed in LPM 1.0.5
